internal/drivers/manager.go

```diff
@@ -555,6 +555,29 @@ func (m *Manager) hibernate(ctx context.Context, poolName, instanceID string) er
 
 	m.waitForInstanceConnectivity(ctx, instanceID)
 
+	retryCount := 1
+	for {
+		err := m.hibernate(ctx, instanceID, poolName, pool)
+		if err == nil {
+			return nil
+		}
+
+		logrus.WithError(err).WithField("retryCount", retryCount).Warnln("failed to hibernate the vm")
+		var re *types.RetryableError
+		if !errors.As(err, &re) {
+			return err
+		}
+
+		if retryCount >= 3 { //nolint:gomnd
```

Comment:
can you fix the lint issue here, rather than ignoring it.

Reply:
Done